### PR TITLE
Jonathan ho

### PR DESCRIPTION
**Pull request by Jonathan-YungHsin-Ho**

### CODE REVIEW

File: Short-Answer/Algorithms_Answers.md [hunk 1]
```diff
@@ -1,15 +1,80 @@
...
+The runtime complexity of this block of code would be `O(n log n)` (linearithmic), where `n` is the input size. This is because the outer `for` loop runs `n` times, for a runtime complexity of `O(n)`. Meanwhile, the value of `j` in the inner `while` loop is doubling rather than incremening, resulting in a runtime complexity of `O(log )`. For nested loops, we want to multiply the runtime complexities, so we would get a combined runtime complexity of `O(n log n)`.
 
 c)
 
+The runtime complexity of this block of code would be `O(n)` (linear), where `n` is `bunnies`. This block of code demonstrates recursion and will run `n` times, decrementing `bunnies` down to the base case value of zero.
+
 ## Exercise II
 
+#### Proposed algorithm:
+
```
Comment (toddmckenzie) [inline Short-Answer/Algorithms_Answers.md:20]: pseudo code looks great for this problem.  Nice job.

File: recursive_count_th/count_th.py [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
-    # TBC
-    
-    pass
+    # # An iterative solution to help conceptualize its corresponding recursive solution:
+    # count = 0
+    # for i in range(0, len(word) - 1):
+    #     if word[i] == 't' and word[i + 1] == 'h':
+    #         count += 1
+    # return count
+
```
Comment (toddmckenzie) [inline recursive_count_th/count_th.py:15]: you could combine line 19 into word[0:2] == 'th'
 you could then call recursive_counter(word[2:], count)
else:
   recurcive_counter(word[1:], count)

File: Short-Answer/Algorithms_Answers.md (hunk 1 above)
Comment (Jonathan-YungHsin-Ho) [reply Short-Answer/Algorithms_Answers.md:20]: Thank you!

File: recursive_count_th/count_th.py (hunk 2 above)
Comment (Jonathan-YungHsin-Ho) [reply recursive_count_th/count_th.py:15]: Great suggestion, thanks!

**Pull request merged**